test/typescript/terraform-cloud/__snapshots__/test.ts.snap

```diff
@@ -3,10 +3,10 @@
 exports[`full integration test deploy in Terraform Cloud 1`] = `
 "Deploying Stack: hello-terra
 Resources
- ✔ NULL_RESOURCE       test                null_resource.test
+ ~ NULL_RESOURCE       test                null_resource.test
```

Comment:
Is this right?

Reply:
No, I hoped that it was fixed with the JSON parsing

Reply:
I'll try a rebase, unsure if it has been lately